fe/src/components/user/user-avatar.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div class="h-[90px] w-[90px]">
+    <img :src="imgSrc" class="rounded-full object-cover h-[90px] w-[90px]" />
+  </div>
+</template>
+
+<script setup lang="ts">
+type Props = {
+  imgSrc: string;
+  width: number;
+  height: number;
```

Comment:
- you did not apply variables to the component. 
- Also it is possible to have default width and height?  so no need to pass again and again for the most used dimension

Reply:
applied the props to component and added default values